lib/mixpanel/client.ex

```diff
@@ -25,6 +25,13 @@ defmodule Mixpanel.Client do
     GenServer.start_link(__MODULE__, {:ok, config}, opts)
   end
 
+  def child_spec([config, opts]) do
```

Comment:
How about, instead of customizing `child_spec/1` we normalize `start_link` to `def start_link([config, opts]) do`...? That would be more idiomatic for a GenServer